examples/toolchains/cc/shell.nix

```diff
@@ -0,0 +1,2 @@
+{ pkgs ? import <nixpkgs> { } }:
```

Comment:
Please pin `nixpkgs`. Otherwise the Bazel version will be floating. Especially important if we want to test this in CI. Otherwise, it's not reproducible.